help.go

```diff
@@ -363,17 +363,17 @@ func printHelpCustom(out io.Writer, templ string, data interface{}, customFuncs
 
 	w := tabwriter.NewWriter(out, 1, 8, 2, ' ', 0)
 	t := template.Must(template.New("help").Funcs(funcMap).Parse(templ))
-	t.New("helpNameTemplate").Parse(helpNameTemplate)
-	t.New("usageTemplate").Parse(usageTemplate)
-	t.New("descriptionTemplate").Parse(descriptionTemplate)
-	t.New("visibleCommandTemplate").Parse(visibleCommandTemplate)
-	t.New("copyrightTemplate").Parse(copyrightTemplate)
-	t.New("versionTemplate").Parse(versionTemplate)
-	t.New("visibleFlagCategoryTemplate").Parse(visibleFlagCategoryTemplate)
-	t.New("visibleFlagTemplate").Parse(visibleFlagTemplate)
-	t.New("visibleGlobalFlagCategoryTemplate").Parse(strings.Replace(visibleFlagCategoryTemplate, "OPTIONS", "GLOBAL OPTIONS", -1))
-	t.New("authorsTemplate").Parse(authorsTemplate)
-	t.New("visibleCommandCategoryTemplate").Parse(visibleCommandCategoryTemplate)
+	_, _ = t.New("helpNameTemplate").Parse(helpNameTemplate)
```

Comment:
lets define a 

`
templates := map[string]string {
  "helpNameTemplate":helpNameTemplate,
....
}
`
and iterate over the loop. Within the loop check the err return value of Parse() and panic/return

Reply:
May effect backward compatability.